tests/responses/test_preset_responses.py

```diff
@@ -0,0 +1,156 @@
+import time
+import pytest
+from flaky import flaky
+from ...gramhopper.responses.preset_responses import _PresetDocumentResponse, \
+    _PresetMessageResponse, _PresetReplyResponse
+
+
+FLAKY_MAX_RUNS = 6
+FLAKY_MIN_PASSES = 1
+
+
+def delay_rerun(*args):  # pylint: disable=unused-argument
+    time.sleep(2)
+    return True
+
+
+@flaky(max_runs=FLAKY_MAX_RUNS, min_passes=FLAKY_MIN_PASSES, rerun_filter=delay_rerun)
+@pytest.mark.usefixtures('bot', 'bot_chat_id', 'generate_new_update')
+class TestPresetMessageResponse:
+
+    SINGLE_PRESET_TEXT = 'one'
+    MULTIPLE_PRESET_TEXTS = ['two', 'three']
+
+    def _test_single_preset_message(self, bot, bot_chat_id, generate_new_update, payload):
+        response = _PresetMessageResponse(self.SINGLE_PRESET_TEXT)
+        update = generate_new_update(chat_id=bot_chat_id)
+
+        message = response.respond(bot, update, payload)
+        assert message
+        assert message.text == self.SINGLE_PRESET_TEXT
+
+    def _test_multiple_preset_messages(self, bot, bot_chat_id, generate_new_update, payload):
+        response = _PresetMessageResponse(self.MULTIPLE_PRESET_TEXTS)
+        update = generate_new_update(chat_id=bot_chat_id)
+
+        message = response.respond(bot, update, payload)
+        assert message
+        assert message.text in self.MULTIPLE_PRESET_TEXTS
+
+    def test_single_preset_message_with_none_payload(self, bot, bot_chat_id, generate_new_update):
```

Comment:
You really should parametrize those test (https://docs.pytest.org/en/latest/example/parametrize.html).
It's quite easy, and the code will be much more readable.